er_example.tsv

```diff
@@ -0,0 +1,3 @@
+	files	files	objects	objects	objects	objects	objects	objects	keys	keys	entities	entities	entities
```

Comment:
I would prefer to keep the root directory relatively clean. Does this need to be part of the repo? If so, can it be placed under `docs` or `tests`?